pkg/controller/ttlafterfinished/ttlafterfinished_controller.go

```diff
@@ -233,7 +241,7 @@ func (tc *Controller) processJob(ctx context.Context, key string) error {
 		PropagationPolicy: &policy,
 		Preconditions:     &metav1.Preconditions{UID: &fresh.UID},
 	}
-	klog.V(4).Infof("Cleaning up Job %s/%s", namespace, name)
+	logger.V(4).Info("Cleaning up Job", "job", klog.KObj(fresh))
```

Comment:
should we have here "fresh" job after final sanity check?

Reply:
`fresh` is valid here because of the error checking above.